ui/react-ui/src/components/routes/userSingleTask/UserSingleTask.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react';
+import Container from 'react-bootstrap/esm/Container';
+import { useParams } from "react-router-dom";
+import Table from 'react-bootstrap/Table';
+
+var UserSingleTask = () => {
+  const [userSingleTask, setUserSingleTask] = useState([]);
+
+  var { userId, taskId } = useParams();
+  console.log(userId, taskId)
+
+  useEffect(() => {
+    fetch(`https://nsc-functionsapp-team1.azurewebsites.net/api/users/${userId}/tasks/${taskId}`)
+    .then(response => response.json())
+    .then(data => setUserSingleTask(data))
+    .catch((error) => console.error(error))
+  }, [userId, taskId]);
+
+  return (
+    <Container>
+      <h3>Selected task</h3>
+      <Table striped bordered hover className="text-left">
```

Comment:
here you could use BootstrapTable component to render the table, it's totally up to you though. if you use that component, it'll have these benefits=>
- you'll avoid hardcoded values
- if in the future some fields get added/removed from the tables, you always get the latest fields.

Reply:
I thought my implementation clashed with how the BootstrapTable component was set up. We could chat more about it if you want

Reply:
don't worry about it, it's already working properly. I'm gonna go ahead and approve you pr.